### PR TITLE
Update intro.md

### PR DESCRIPTION
**Pull request by the-wazz**
Added a missing step (and some text) to the 'Add Razor Pages using Entity Framework (CRUD)' dialog:

### CODE REVIEW

File: aspnetcore/data/ef-rp/intro.md [hunk 1]
```diff
@@ -221,7 +221,8 @@ In this section, you use the ASP.NET Core scaffolding tool to generate:
   * In the **Model class** drop-down, select **Student (ContosoUniversity.Models)**.
   * In the **Data context class** row, select the **+** (plus) sign.
     * Change the data context name to end in `SchoolContext` rather than `ContosoUniversityContext`. The updated context name: `ContosoUniversity.Data.SchoolContext`
-   * Select **Add**.
+    * Select **Add** to finish adding the Data context class.
+   * Select **Add** to finish the 'Add Razor Pages' dialog.
```
Comment (scottaddie) [inline aspnetcore/data/ef-rp/intro.md:225]:
```suggestion
    * Select **Add** to finish adding the data context class.
   * Select **Add** to finish the **Add Razor Pages** dialog.
```

**Review by scottaddie — CHANGES_REQUESTED** [review]
Thank you for the PR. Here are some minor changes to incorporate.

**Comment by the-wazz** [comment]
@Rick-Anderson @scottaddie My first time incorporating someone else's changes. Let me know if I did it wrong or there's something else to do.

**Review by scottaddie — APPROVED** [review]

**Comment by scottaddie** [comment]
@the-wazz You did great! Thank you for fixing this issue in the doc.

**Pull request merged**